python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -188,11 +191,11 @@ def makeIdFactory(self, dataRef):
                              (e.g. visit, raft, sensor for LSST data).
         """
         expBits = dataRef.get("ccdExposureId_bits")
-        expId = long(dataRef.get("ccdExposureId"))
+        expId = int(dataRef.get("ccdExposureId"))
```

Comment:
Great catch. I though future had its own version of int to handle this, but I guess not. Ouch!